Reference_Manual.md

```diff
@@ -1202,114 +1209,111 @@ You can also fine-tune this feature using `client_config::set_protperty` API as
+```c++
+    hazelcast::client::client_config config;
+
+    boost::asio::ssl::context ctx(boost::asio::ssl::context::method::tlsv12_client);
+    ctx.set_verify_mode(boost::asio::ssl::verify_peer);
+    ctx.set_default_verify_paths();
     
-    /**
-     * @return Returns the use configured cipher list string.
-     */
-    const std::string &get_cipher_list() const;
+    // This config is to validate the server certificate if server does not have a CA signed certificate
+    ctx.load_verify_file("/path/to/my/server/public/certificate");
     
-    /**
-     * @param ciphers The list of ciphers to be used. During client start, if this API was set then the
-     * SSL_CTX_set_cipher_list (https://www.openssl.org/docs/man1.0.2/ssl/SSL_set_cipher_list.html) is
-     * called with the provided ciphers string. The values and the format of the ciphers are described here:
-     * https://www.openssl.org/docs/man1.0.2/apps/ciphers.html Some examples values for the string are:
-     * "HIGH", "MEDIUM", "LOW", etc.
-     *
-     * If non of the provided ciphers could be selected the client initialization will fail.
-     *
-     */
-    ssl_config  &set_cipher_list(const std::string &ciphers);
-```
+    // The following two lines configure the client to use the client certificate to introduce itself to the server
+    ctx.use_certificate_file("/path/to/my/client/public/certificate", boost::asio::ssl::context::pem);
+    ctx.use_private_key_file("/path/to/my/client/private/certificate", boost::asio::ssl::context::pem);
 
-You can set the protocol as one of the following values:
+    config.get_network_config().get_ssl_config().set_context(std::move(ctx));
+    auto hz = hazelcast::new_client(std::move(config)).get();
 ```
-sslv2, sslv3, tlsv1, sslv23, tlsv11, tlsv12
+
+With the above configuration, both the client and the server authenticate each other mutually. The client validates server certificate and the server validates the client with the client certificate.
+
+You can check `mutual_authentication.cpp` example for a full featured example.
+
+#### 6.1.2.3. Constraining The Used Cipher List
+In some cases, you may want to limit the cipher suites allowed for a client to while communicating the server. This can also be configured using the `ssl_config`. Here is an example configuration to enable cipher list which only allow "HIGH" ciphers.
```

Comment:
... for a client while ... 

... configured using `ssl_config`.

... which allows only "HIGH" ...